app/screens/search/search.js

```diff
@@ -92,7 +92,7 @@ class Search extends Component {
     componentDidUpdate() {
         const {searchingStatus, recent} = this.props;
         const recentLenght = recent.length;
-        const shouldScroll = searchingStatus === RequestStatus.SUCCESS || searchingStatus === RequestStatus.STARTED;
+        const shouldScroll = (searchingStatus === RequestStatus.SUCCESS || searchingStatus === RequestStatus.STARTED) && !this.recentRemoved;
```

Comment:
Shouldn't we trigger this scroll based on the previous props instead of needing this extra flag for this one case? I feel like any other state change will cause it to scroll randomly. Like we should only scroll when `prevStatus !== status && (status === SUCCESS || status === STARTED)`?